src/app/MessageDef.cpp

```diff
@@ -2631,6 +2631,135 @@ AttributeDataList::Builder & AttributeDataList::Builder::EndOfAttributeDataList(
     return *this;
 }
 
+#if CHIP_CONFIG_IM_ENABLE_SCHEMA_CHECK
```

Comment:
QQ: is this file auto-generated or did you manually add this code?

If this is manually, I have concerns about adding more stuff to a file that got to 3800+ lines of code. Is there something we can do for maintainability?

Reply:
manually, i will handle the refactor here in next PR, it should happen next week.
https://github.com/project-chip/connectedhomeip/issues/3644

Reply:
per offline chat, will do the refactor #3644 in this pr

Reply:
Done, thanks